app/Alexa/Models/Role.php

```diff
@@ -6,6 +6,21 @@
 
 class Role extends Model
 {
+    /**
+     * Administrator role type.
+     *
+     * @var string
+     */
+    const ROLE_ADMINISTRATOR = 'admin';
+    const ROLE_USER = 'user';
+
+    /**
+     * Indicates if model should be timestamped.
+     *
+     * @var bool
+     */
+    public $timetamps = false;
```

Comment:
Typo. `$timestamps` (the origin of tests failure)

Reply:
I was looking for that 👍